pdc_client/plugins/auth_group_resource_permissions.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+from __future__ import print_function
+
+import json
+
+from pdc_client.plugin_helpers import (PDCClientPlugin,
+                                       extract_arguments,
+                                       add_parser_arguments,
+                                       add_create_update_args)
+
+
+class AuthGroupResourcePermissionPlugin(PDCClientPlugin):
+    def register(self):
+        self.set_command('auth/group-resource-permissions')
```

Comment:
It is interesting for the command name auth/group-resource-permissions, it looks like in the future there are other commands in auth, like auth/others. Is it ok to just set command to group-resource-permissions. But of course auth/group-resource-permissions is workable.